operators.md

```diff
@@ -0,0 +1,175 @@
+# Currently supported operations
+
+| Operation type            | Support            | Opset versions     |
```

Comment:
1) 'Support' -> 'Supported'
2) In 'Support' column keep only Y/N status
3) Column 'Opset version' can be dropped. You should write minimal tested (9), maximum tested (15) and recomended (13) opset versions before table.  
4) Please add new column with restriction notes like in nvvidia repository. In that column you must write about padding restrinctions (conv operation)...